tools/lib/NQP/Config/Rakudo.pm

```diff
@@ -665,7 +662,7 @@ sub gen_nqp {
 
     return unless defined($gen_nqp) || defined($gen_moar);
 
-    if ( defined $gen_nqp || defined $gen_moar ) {
+    {
         my $user = $options->{'github-user'} // 'perl6';
```

Comment:
BTW, isn't it time to replace _'perl6'_ with _'Raku'_?

Reply:
Done.